lib/msp432/e4/systemcontrol.c

```diff
@@ -0,0 +1,130 @@
+/** @defgroup systemcontrol_file System Control
+ *
+ * @ingroup MSP432E4xx
+ *
+ * @brief libopencm3 MSP432E4xx System Control
+ *
+ * @version 1.0.0
+ *
+ * @date 22 July 2018
+ *
+ * LGPL License Terms @ref lgpl_license
+ */
+
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2012 Alexandru Gagniuc <mr.nuke.me@gmail.com>
```

Comment:
did you keep anything from the?  I can't really see it, but if you think it's worthwhile, sure, keep it.

Reply:
During development, I relied heavily on systemcontrol from lm4f. So, I want keep it.